kolibri/content/api.py

```diff
@@ -143,9 +106,20 @@ def get_missing_files(channel_id=None, content=None, **kwargs):
     """
     if content.kind == 'topic':
         all_end_nodes = leaves(channel_id=channel_id, content=content)
-        return KolibriContent.File.objects.using(channel_id).filter(available=False, format__contentmetadata__in=all_end_nodes)
+        return KolibriContent.File.objects.using(channel_id).filter(available=False, contentmetadata__in=all_end_nodes)
```

Comment:
maybe when people download resource for a content, this api will return what need to be downloaded.

If supplementary also get returned, the front end can render them in a way the user know they are optional.